### PR TITLE
abjad 3.4 (new formula)

### PR DESCRIPTION
**Pull request by nwhetsell**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

This PR adds [Abjad](https://abjad.github.io), a Python API for building [LilyPond](https://lilypond.org) files.

### CODE REVIEW

File: Formula/abjad.rb [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    sha256 "25bfab43b5d67e50762b21da49fc7d8439406b7ca31b063f3e34299949f1a5e6"
+  end
+
+  def install
+    virtualenv_install_with_resources
+
+    python_version = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
+    (lib/"python#{python_version}/site-packages/homebrew-abjad.pth").write <<~EOS
+      import site; site.addsitedir('#{libexec}/lib/python#{python_version}/site-packages')
+    EOS
```
Comment (carlocab) [inline Formula/abjad.rb:85]: We usually don't package Python libraries (see [Python for Formula Authors](https://docs.brew.sh/Python-for-Formula-Authors)) -- especially not when you have to write a `.pth` file for them (See #73875).

There are some exceptions, but most, if not all, of them have no Python `resource`s as dependencies.

**Review by carlocab — COMMENTED** [review]
Thanks for the PR, but I'm not sure this is a good fit for Homebrew/core. (See below.)

File: Formula/abjad.rb (hunk 1 above)
Comment (nwhetsell) [reply Formula/abjad.rb:85]: Oops, missed that, sorry for the noise!

Comment (carlocab) [reply Formula/abjad.rb:85]: No worries! Thanks for taking the time to contribute anyway.

**Pull request closed without merging**